lib/engine_cart/configuration.rb

```diff
@@ -68,12 +68,13 @@ def verbose?
     private
 
     def default_config
+      environment_options = ENV['ENGINE_CART_RAILS_OPTIONS']
       {
         engine_name: ENV['CURRENT_ENGINE_NAME'] || default_engine_name,
         destination: ENV['ENGINE_CART_DESTINATION'] || ENV['RAILS_ROOT'] || default_destination,
         template: ENV['ENGINE_CART_TEMPLATE'],
         templates_path: ENV['ENGINE_CART_TEMPLATES_PATH'] || './spec/test_app_templates',
-        rails_options: ENV['ENGINE_CART_RAILS_OPTIONS']
+        rails_options: environment_options ? environment_options.split(/\s+/) : nil
```

Comment:
Sure, but I could set `ENGINE_CART_RAILS_OPTIONS="-d postgres"`, say. I don't know how the generator arguments are supposed to work, so maybe it's fine to split them all up..